processor/stash/stash.go

```diff
@@ -40,8 +40,8 @@ func Init() {
 
 func loadConfig() {
 	config.RegisterBoolConfigVariable(true, &errorStashEnabled, true, "Processor.errorStashEnabled")
-	config.RegisterDurationConfigVariable(30, &errReadLoopSleep, true, time.Second, []string{"Processor.errReadLoopSleep", "errReadLoopSleepInS"}...)
-	config.RegisterIntConfigVariable(1000, &errDBReadBatchSize, true, 1, "Processor.errDBReadBatchSize")
+	config.RegisterDurationConfigVariable(5, &errReadLoopSleep, true, time.Second, []string{"Processor.errReadLoopSleep", "errReadLoopSleepInS"}...)
+	config.RegisterIntConfigVariable(10000, &errDBReadBatchSize, true, 1, "Processor.errDBReadBatchSize")
```

Comment:
We can keep the current numbers and skip sleeping for `errReadLoopSleep` until the stash loop picks up less jobs than the `errDBReadBatchSize`